### DISCUSSION TITLE
記事内の画像に枠がつく新機能は、全然欲しくない。

### DISCUSSION THREAD

**Discussion by oci-tech** [increments/qiita-discussions#258] [要望・アイデア・不具合]
2022-06-02、「記事内の画像に枠がつく」という機能ができました。
最初、アナウンスを気づかず、画像を見たらあれっ、なんか変だなと思っていました。
よく見たら、細い枠があります。正直にこの機能すごく嫌です。
前の画像は、よっぽどきれいでしたのに。。。
枠のせいで、見た目が悪くなった印象です。

もちろん、この機能が好きな方が、少なくないと思います。
みんなそれぞれの好みがありますので、せめて、この機能がいるか要らないか選択肢がほしいです。

**Comment by tanaka-asami415** [comment]
@liu-wei1 
ご意見ありがとうございます🙏
枠の有無の切替については、お気持ちはとてもわかるのですが、直近での実現はかなり難しそうです🙏
>見た目が悪くなった印象です。

こちらは何か別の形で改善できないか、検討させていただきます🙇
